cvat/apps/engine/utils.py

```diff
@@ -357,6 +359,29 @@ def take_by(iterable: Iterable[_T], chunk_size: int) -> Generator[list[_T], None
         yield batch
 
 
+def get_paths_sizes(paths: list[str]) -> dict[str, int | OSError | ValueError]:
+    sizes: dict[str, int | Exception] = {}
+    for path in paths:
```

Comment:
As far as I can see, there's no reason for this function to take a list of paths at this point. It should just take a path and return `int` (or raise an exception). It would make both the function and the calling code simpler.